hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/HtsSystemContract.java

```diff
@@ -113,11 +113,7 @@ private static FullResult resultOfExecuting(
                             .systemOperations()
                             .externalizeResult(
                                     contractFunctionResultSuccessFor(
-                                            pricedResult.fullResult().gasRequirement(),
-                                            output,
-                                            frame.getRemainingGas(),
-                                            frame.getInputData(),
-                                            attempt.senderId()),
+                                            pricedResult.fullResult().gasRequirement(), output, attempt.senderId()),
```

Comment:
⚠️ We sometimes want to include `gas`, `amount`, `functionParameters`; and sometimes not. (See mono-service [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/precompile/Precompile.java#L118) and [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/precompile/utils/PrecompileUtils.java#L57.).)

This was fixed in [this PR](https://github.com/hashgraph/hedera-services/commit/a05019567c0abdb58003f9bfe8b3ba359686eb09).

You should be able to revert this change as well as in `SystemContractUtils`, merge `develop` and this should pass.